sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/download/manager/DownloadManagerReleaseDownloaderTest.java

```diff
@@ -449,12 +454,30 @@ public void errorOnOpenDownloadedFile() throws IOException {
         verify(mListener).onError(anyString());
     }
 
+    @Test
+    public void errorOnCloseFileDescriptor() throws IOException {
```

Comment:
what is this test for?

Reply:
In order to cover all error cases just in case